src/main/java/com/farao_community/farao/flow_decomposition/FlowDecompositionResults.java

```diff
@@ -26,8 +26,11 @@
  * @see DecomposedFlow
  */
 public class FlowDecompositionResults {
-    private final boolean saveIntermediates;
-    private final String id;
+    static final boolean FILL_ZEROS = true;
+    static final boolean NOT_FILL_ZEROS = false;
+    private static final boolean DEFAULT_FILL_ZEROS = NOT_FILL_ZEROS;
+    private final FlowDecompositionParameters parameters;
```

Comment:
Keep saveIntermediates and initialises with parameters.saveIntermediates

Reply:
ok